src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -125,6 +126,10 @@ public Keyspaces apply(ClusterMetadata metadata)
         if (!table.params.compression.isEnabled())
             Guardrails.uncompressedTablesEnabled.ensureEnabled(state);
 
+        if (table.params.transactionalMode.accordIsEnabled && !DatabaseDescriptor.getAccordTransactionsEnabled())
```

Comment:
there's not, do you think we need a config option?

Reply:
I think it's pretty important yeah since otherwise you have to be very careful when creating schema and if you aren't you have to run the migration process. If the migration process were turn key it would be much less of an issue.